### DISCUSSION TITLE
Audio Input and recording

### DISCUSSION THREAD

**Discussion by WhiteWolf47** [TomSchimansky/CustomTkinter#1827] [Q&A]
I want to create a customtkinter gui to transcribe audio to text, for this i want to add the option to upload audio and also record audio to transcribe them, but i have no idea how can i add the option to upload audio and record audio in customtkinter?

**Comment by avalon60** [comment]
This doesn't directly impinge on anything much CustomTkinter specifically,  other than possibly the display or the output, for which you would likely use CTkTextBox. For example, you would probably want to use the Tkinter, askopenfilename(), widget to select an audio file. 
For the most part it's not CustomTkinter related:
https://pythonbasics.org/transcribe-audio/
https://realpython.com/playing-and-recording-sound-python/
